### DISCUSSION TITLE
Is there a way to get custom sub scene?

### DISCUSSION THREAD

**Discussion by chengenzhao** [AlmasB/FXGL#1212] [Q&A] [answered]
Hi:

Since this [issue](https://github.com/AlmasB/FXGL/issues/995) has been solved
FXGL currently allow developers to create custom sub scenes
but how to find the sub scene?
Is there a method like: 
```java
FXGL.getSceneService().getSubSceneByID(name: "MyCustomSubScene", id:32167);
```
?

Thanks

**Comment by AlmasB** [comment, marked as answer]
Hi, no such method. The easiest way is to keep a reference to the custom subscene since they are owned by the developer.
